### PR TITLE
imp: request, response spec update

### PR DESCRIPTION
**Pull request by jyc228**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module kroma-prover-proxy
```
Comment (0xHansLee) [inline go.mod:1]: Please change module name to `github.com/kroma-network/kroma-prover-grpc-proto`

Comment (jyc228) [reply go.mod]: applied

**Review by 0xHansLee — APPROVED** [review]
lgtm

**Review by dongchangYoo — APPROVED** [review]
LGTM

**Review by lightscale-luke — APPROVED** [review]

**Comment by lightscale-luke** [comment]
please update commit head
```
feat: update request and response spec
```

**Pull request merged**